src/vivarium/framework/state_machine.py

```diff
@@ -18,7 +18,7 @@
 if TYPE_CHECKING:
     from vivarium.framework.engine import Builder
     from vivarium.framework.population import PopulationView
-    from vivarium.framework.time import Time
+    from vivarium.types import Time
```

Comment:
What is happening here that we now need to import Time like this?

Reply:
@rmudambi Oh shoot, I forgot to flag this. I moved all shared custom types into a new types.py module so that I can use ScalarValue more intuitively (instead of needing to remember it's in lookup.py). And then I forgot to actually use ScalarValue.